var/spack/repos/builtin/packages/rocm-validation-suite/package.py

```diff
@@ -111,7 +113,11 @@ class RocmValidationSuite(CMakePackage):
     patch("006-library-path.patch", when="@4.5.0:5.2")
     patch(
         "007-cleanup-path-reference-donot-download-googletest-yaml-library-path_5.3.patch",
-        when="@5.3.0:",
+        when="@5.3.0:5.5",
+    )
+    patch(
```

Comment:
is this patch applicable for 5.6.1?

Reply:
This patch will be applied for all 5.6 version including 5.6.1 when it is "@5.6".
However updating as "@5.6:" to extend it till the changes are available in the upstream code.